tools/mslite

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -e
+pushd $(cd $(dirname ${0})/..; pwd) > /dev/null
+
+bold() {
+    echo "$(tty -s && tput bold)$1$(tty -s && tput sgr0)"
+}
+
+clean() {
+    bold "mindspore clean"
+    rm -rf "./third_party/source/mindspore"
+}
+
+sync() {
+    bold "mindspore sync"
+    mkdir -p "./third_party/source/mindspore/"
+    curl --silent --location --output "./third_party/source/mindspore/model.fbs" "https://github.com/mindspore-ai/mindspore/blob/master/mindspore/lite/schema/model.fbs?raw=true"
+    curl --silent --location --output "./third_party/source/mindspore/ops.fbs" "https://github.com/mindspore-ai/mindspore/blob/master/mindspore/lite/schema/ops.fbs?raw=true"
+}
+
+schema() {
+    echo "mslite schema"
+
+    # modify
+    cp ./third_party/source/mindspore/*.fbs ./tools/
+    sed -i '1,16d' ./tools/model.fbs
```

Comment:
Please remove all `sed` commands and fix either the original `.fbs` schema file or `mslite.js`.